### PR TITLE
Ta test 41244

### PR DESCRIPTION
**Pull request by ApurwaMohite**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/dtm/test_single_process_restart.py [hunk 1]
```diff
@@ -110,6 +117,8 @@ def teardown_method(self):
             resp = support_bundle_utils.collect_support_bundle_k8s(local_dir_path=path,
                                                                    scripts_path=K8S_SCRIPTS_PATH)
             assert_utils.assert_true(resp)
+        if not os.path.exists(self.test_dir_path):
```
Comment (sonalk0209) [inline tests/dtm/test_single_process_restart.py:120]: *if os.path.exists
"not" not needed

File: tests/dtm/test_single_process_restart.py [hunk 2]
```diff
@@ -347,3 +356,80 @@ def test_bkt_creation_ios_after_m0d_restart(self):
...
+        self.log.info("Step 3: Perform Single m0d Process Restart")
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=POD_NAME_PREFIX,
+                                            container_prefix=MOTR_CONTAINER_PREFIX,
+                                            process=self.m0d_process, check_proc_state=True)
+        assert_utils.assert_true(resp, "Failure in observed during process restart/recovery")
+        self.log.info("Step 3: m0d restarted and recovered successfully")
+
+        self.log.info("Step 4: Download the uploaded object after m0d recovery and verify checksum")
```
Comment (sonalk0209) [inline tests/dtm/test_single_process_restart.py:422]: so the purpose of the test to download uploaded object and verify checksum and not uploading another object? @ApurwaMohite  @priyankaborawake

Comment (ApurwaMohite) [reply tests/dtm/test_single_process_restart.py:422]: Yes, the intention is multiparts uploaded before m0d restart and consistent post recovery also.
@priyankaborawake please confirm once.

**Review by kulrk — APPROVED** [review]

File: tests/dtm/test_single_process_restart.py (hunk 2 above)
Comment (priyankaborawake) [reply tests/dtm/test_single_process_restart.py:620]: @ApurwaMohite @sonalk0209  Yes, this test was scoped to check if written multipart object can be read post restart. It was conveyed multipart operation are not supported during/post restart.

**Review by priyankaborawake — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**